bedrock/base/templates/includes/structured-data/organization/mozilla-foundation-organization.json

```diff
@@ -0,0 +1,31 @@
+{
+    "@context": "http://schema.org",
+      "@type": "NGO",
+      "@id": "https://foundation.mozilla.org/#organisation",
+      "name": "Mozilla Foundation",
+      "url": "https://foundation.mozilla.org",
+      "sameAs": [
+        "https://www.wikidata.org/wiki/Q55672",
+        "https://foundation.mozilla.org",
+        "https://en.wikipedia.org/wiki/Mozilla_Foundation"
+      ],
+      "subOrganization": {
+        "@type": "Organization",
+		"@id": "https://www.mozilla.org/#organization",
```

Comment:
nit: indenting is a bit off in some of these files re: tabs-vs-spaces.
You could try <kbd>npx prettier ./bedrock/base/templates/includes/structured-data/**/*.json --write</kbd> to auto-format the JSON files, but there are a couple errors in some of the files.